pkg/services/provisioning/datasources/config_reader_test.go

```diff
@@ -149,6 +150,12 @@ func TestDatasourceAsConfig(t *testing.T) {
 			So(err, ShouldNotBeNil)
 		})
 
+		Convey("invalid access should warn about invalid value and return 'proxy'", func() {
+			reader := &configReader{log: logger}
+			configs, _ := reader.readConfig(invalidAccess)
```

Comment:
Why do you ignore the error here, instead of verifying it's nil?

Reply:
My bad. I will fix it.